admin/helpers/hdf-presenter.js

```diff
@@ -11,4 +11,26 @@ hdfPresenter.getCanViewResults = (resultsDate) => {
   return moment().utc().isAfter(resultsDate)
 }
 
+hdfPresenter.getPupilsWithViewStatus = (pupils) => {
```

Comment:
Would be nice to have some comments about what this function does, and when to use it.  This seems like a partial view of process status, minus the restart, and developed just for this use.

Reply:
@jon-shipley added jsdoc to provide explanation.